devices.js

```diff
@@ -1,11 +1,17 @@
 const waveshare4In2Driver = require('bindings')('waveshare4in2.node');
+const waveshare2In7Driver = require('bindings')('waveshare2in7.node');
 
 const devices = {
-    waveshare4in2: {
-        height: 300,
-        width: 400,
-        driver: waveshare4In2Driver
-    }
-}
+  waveshare4in2: {
+    height: 300,
+    width: 400,
+    driver: waveshare4In2Driver,
+  },
+  waveshare2in7: {
+    height: 264,
+    width: 176,
+    driver: waveshare2In7Driver,
```

Comment:
Same vales from `c/EPD_2in7.h` file

Reply:
This is the correct place to get dimensions, I wonder if they're not getting passed around correctly (for some reason). See my comment below.

Reply:
Thanks for the quick response! 

I added the log, and i get the expected `[264, 176]` output.

Reply:
Could it be just that the fonts are too big? I'm not super familiar with puppeteer, but is there maybe a way to save the 1bit image to the disk before displaying it ?

Reply:
A couple other things to try:
1. put a console log in page.js:26 and make path points to waveshare2in7.node
```
console.log(this.screen.driver);
```

2. In weather.js, instead of:
```
const {init} = require('../../index.js');
const devices = require('../../devices.js')
```
Revert back to 
```
const {init} = require('epaperjs');
```
and then do the following in the examples/weather-station directory:
```
rm -rf node_modules
npm install
```
this should force a clean rebuild of the waveshare drivers and the full epaper module

Reply:
> Could it be just that the fonts are too big? I'm not super familiar with puppeteer, but is there maybe a way to save the 1bit image to the disk before displaying it ?

I tried opening raspberrypi:3000 in a web browser on my laptop and setting the size to 264x176 and everything seemed to scale down, correctly. But you could try just creating a div with a fixed size and black border, or something, to test.

Reply:
> console.log(this.screen.driver);

It points correctly to `/home/pi/epaper.js/build/Release/waveshare2in7.node`

> `const {init} = require('epaperjs');`

Ah I didn't know that we would still be using the local version! I thought this would be loaded from NPM.

I tried with a chrome browser set to those dimensions as well, the page seems to render correctly.

I'll try to investigate further. Thanks for all the help

Reply:
It actually looks to be a css issue. I was able to reproduce the issue by doing the following:
1. Open raspberrypi:3000 in chromium on another computer
2. Open dev tools, toggle device toolbar, set dimensions to 176x264
3. Refresh page

After the refresh, I saw the the following:
![Screen Shot 2020-08-26 at 12 50 28 PM](https://user-images.githubusercontent.com/13860662/91332882-cf78c600-e79a-11ea-82f7-01b27ac9e4dd.png)

Reply:
This seems to work 🥳  !


```html 
   <style>
        body {
          margin: 0;
        }
        #box {
          width: calc(100vw - 2px);
          height: calc(100vh - 2px);
          border: 1px solid black;
          font-size: 24px;
        }
      </style>
	 <div id="box">
            Hello world!
     </div>
```


![IMG_20200826_185028](https://user-images.githubusercontent.com/1442690/91332940-2bf4d900-e7cd-11ea-842c-fcfbcdaefb21.jpg)

Reply:
Try adding the following css to the bottom of the styles in the weather-example index.html
``` css
@media only screen and (max-width:176px) {
  html {
    font-size: 8px;
  }
}
```

Reply:
Amazing! yes that was it !

![IMG_20200826_185708](https://user-images.githubusercontent.com/1442690/91333542-ff8d8c80-e7cd-11ea-898e-02239bdbde38.jpg)

Reply:
Awesome! I should have some time in the next day or two to look at rotation!